front/src/components/search/index.tsx

```
@@ -111,6 +99,17 @@ const Search = (): React.ReactElement => {
     setIsLoading(false);
   };
 
+  const executeCountQuery = async (fhirUrl: string) => {
+    let countUrl = new URL(fhirUrl);
+    countUrl.searchParams.append('_summary', 'count')
+    try {
+      const response: any = await axios.get(countUrl.toString());
+      setTotalCount(response.data.total);
+    } catch (err) {
+      return;
```

swallowing errors like this makes it hard to debug sometimes, don't you want to console.error() it at least ?